django/thunderstore/modpacks/api/experimental/tests/test_legacyprofile.py

```diff
@@ -0,0 +1,80 @@
+import io
+import json
+
+import pytest
+from django.core.files import File
+from django.urls import reverse
+from rest_framework.test import APIClient
+
+from thunderstore.modpacks.models import LegacyProfile
+from thunderstore.modpacks.models.legacyprofile import LEGACYPROFILE_STORAGE_CAP
+
+
+@pytest.mark.django_db
+def test_experimental_api_legacyprofile_create(api_client: APIClient) -> None:
+    assert LegacyProfile.objects.count() == 0
+    test_content = b"hunter2"
+    response = api_client.post(
+        reverse("api:experimental:legacyprofile.create"),
+        data=test_content,
+        content_type="application/octet-stream",
+    )
+    assert response.status_code == 200
+    result = json.loads(response.content.decode())
+    assert "key" in result
+    assert LegacyProfile.objects.count() == 1
+    obj = LegacyProfile.objects.get(id=result["key"])
+    assert obj.file.read() == test_content
+    assert LegacyProfile.objects.filter(id=result["key"]).exists()
```

Comment:
Does this test anything that the line 26 didn't already test?

Reply:
It doesn't, forgot to delete this line, removed